ui/raidboss/emulator/ui/EncounterTab.ts

```diff
@@ -55,8 +55,8 @@ export default class EncounterTab extends EventBus {
         const zone = enc.zoneName;
         // ?? operator here to account for old encounters that don't have the property
         const encDate = DTFuncs.timeStringToDateString(enc.start, enc.tzOffsetMillis ?? 0);
-        const encTime = DTFuncs.timeToTimeString(enc.start, enc.tzOffsetMillis ?? 0);
-        const encDuration = DTFuncs.msToDuration(enc.duration);
+        const encTime = DTFuncs.timeToTimeString(enc.start + enc.offset, enc.tzOffsetMillis ?? 0);
+        const encDuration = DTFuncs.msToDuration(enc.duration - enc.offset);
```

Comment:
Do you think it'd make sense to have a `PersistorEncounter` getter property for `trimmedDuration` and `trimmedStart` (or some other name)?

Reply:
I'll just merge this patch for now and this can be a followup if I or you are interested in it.